docs/coverage-reporter/index.md

```diff
@@ -423,6 +423,14 @@ Follow these instructions to validate that your coverage setup is working correc
             Check <a href="../repositories-configure/ignoring-files/">which files are ignored on Codacy</a> and make sure that you're generating coverage reports for the correct files in your repository.
         </td>
     </tr>
+    <tr>
+        <td>
+            The uploaded coverage data is incorrectly marked, using the `-l` option, as only including files in a language that's not present in your repository.
```

Comment:
Small suggestion: instead of using the term "marked" we could refer to the coverage data being "associated" with a language. For example:

```suggestion
            The uploaded coverage data is incorrectly associated with the `-l` option to a language that's not present in your repository.
```